pkg/middleware/requestmeta/request_metadata.go

```diff
@@ -0,0 +1,75 @@
+package requestmeta
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/grafana/grafana/pkg/web"
+)
+
+const (
```

Comment:
maybe it makes sense to somehow unify those lists?

Reply:
with https://github.com/grafana/grafana/blob/8ec4c1bdc87c43fe43e6bc56db924a9107edf137/pkg/services/featuremgmt/codeowners.go

Reply:
I was thinking about that but codeowners are not really the same since these will be used to route alerts to on call schedules and multiple code owners might be part of one schedule. 

Long term it might make sense to move both these consts and the code owner consts to some kind of code meta package. But lets do that later :)

Reply:
+1 that's what I came here to say. We should abstracts teams to a single enum that we attach escalation policies various metadata to that we can reuse across packages for whatever the use case might be